Point to versioned samples for appsettings_SQLite.json connection string (#21142)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/razor-pages/sql.md

# [Visual Studio](#tab/visual-studio)

[!code-csharp[](razor-pages-start/sample/RazorPagesMovie50/Startup.cs?name=snippet_ConfigureServices&highlight=5-6)]

# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)

[!code-csharp[](razor-pages-start/sample/RazorPagesMovie50/Startup.cs?name=snippet_UseSqlite&highlight=5-6)]

---

The ASP.NET Core [Configuration](xref:fundamentals/configuration/index) system reads the `ConnectionString` key. For local development, configuration gets the connection string from the *appsettings.json* file.

# [Visual Studio](#tab/visual-studio)

> The generated connection string will be similar to the following example:

Reviewer: NO FT
```suggestion
The generated connection string is similar to the following JSON:
```
